data/download_data.py

```diff
@@ -248,6 +256,64 @@
         'compressed': False,
         'desc': 'Reader weights trained on Trivia multi hybrid retriever results and HF bert-base-uncased model'
     },
+ 
+    # extra checkpoints for EfficientQA competition
+    'checkpoint.reader.nq-single-seen_only.hf-bert-base': {
+        's3_url': 'https://dl.fbaipublicfiles.com/dpr/checkpoint/reader/nq-single-seen_only/hf_bert_base.cp',
+        'original_ext': '.cp',
+        'compressed': False,
+        'desc': 'Reader weights trained on NQ-single retriever results and HF bert-base-uncased model, when only Wikipedia pages seen during training are considered'
+    },
+    'checkpoint.reader.nq-drqa.hf-bert-base': {
+        's3_url': 'https://dl.fbaipublicfiles.com/dpr/checkpoint/reader/nq-drqa/hf_bert_base.cp',
+        'original_ext': '.cp',
+        'compressed': False,
+        'desc': 'Reader weights trained on DrQA results and HF bert-base-uncased model'
+    },
+    'checkpoint.reader.nq-drqa-seen_only.hf-bert-base': {
+        's3_url': 'https://dl.fbaipublicfiles.com/dpr/checkpoint/reader/nq-drqa-seen_only/hf_bert_base.cp',
+        'original_ext': '.cp',
+        'compressed': False,
+        'desc': 'Reader weights trained on DrQA results and HF bert-base-uncased model, when only Wikipedia pages seen during training are considered'
+    },
+ 
+    # retrieval indexes
+    'checkpoint.indexes.single.nq.full-index-dpr': {
```

Comment:
May be starting with "indexes." is better? "checkpoint" feels like some model weights checkpoint